### ISSUE TITLE
Streamlined publishing 

### ISSUE THREAD

**Issue by jamie9090** [oleeskild/obsidian-digital-garden#209]
Feature request:

From a UX perspective - it would be awesome to add a command for "Publish and share" where it adds a public tag to the front matter, publishes to Vercel and then copies note URL to clipboard!

**Comment by oleeskild** [comment]
Good idea! I've released a version 2.40.0 of the plugin which add the command "Quick Publish And Share Note", which does the three things you describe here.
